internal/pkg/diff/diff.go

```diff
@@ -168,10 +168,6 @@ func (d *Differ) newOptions() cmp.Options {
 		cmpopts.AcyclicTransformer("strByLine", func(s string) []string {
 			return strings.Split(s, "\n")
 		}),
-		// Compare only relations defined on the API side
```

Comment:
Tu som sa trocha sekol.
Diff sa pouziva aj pri ukladani `local` stavu, a preto tam musia byt vsetky `relations`, nie iba API side, ale aj manifest side.

Ak by som to nezmazal, ... tak ak pri `pull` dostane nejaky existujuci objekt novu `Relation`, tak sa neulozi do manifestu, pretoze objekt nie je v diffe.